### DISCUSSION TITLE
How to list all the repositories

### DISCUSSION THREAD

**Discussion by Divyachilukuri-kaplan** [community/community#61605] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I just want to make note of all repositories in my account and i want to check the merged and unmerged branches without cloning the repo in to local machine

**Comment by nirav-gajera** [comment]
1. Go to the GitHub: https://github.com/ website and sign in to your account.
2. Click on your username in the top right corner of the page.
3. Select "Your repositories" from the dropdown menu.
4. This will show you a list of all of your repositories.
5. To check the merged and unmerged branches for a specific repository, click on the name of the repository.
6. This will open the repository in a new tab.
7. On the left side of the page, you will see a list of branches.
8. The branches that are merged will have a green checkmark next to them.
9. The branches that are unmerged will not have a checkmark next to them.

You can also use the following Git commands to check the merged and unmerged branches for a repository:

```
git ls-remote <remote_url>
```

This will list all of the branches that are tracked by the remote repository.

```
git fetch <remote_name>
```

This will fetch the latest changes from the remote repository.

```
git branch -a
```

This will list all of the branches in the local repository, including the merged and unmerged branches.

**Comment by Divyachilukuri-kaplan** [comment]
i want to have all repositories in an organization and in a document format
